storage/pebble/blocks.go

```diff
@@ -14,40 +14,22 @@ import (
 
 var _ storage.BlockIndexer = &Blocks{}
 
-type BlockOption func(block *Blocks) error
-
-// WithInitHeight sets the first and last height to the provided value,
-// this should be used to initialize an empty database, if the first and last
-// heights are already set an error will be returned.
-func WithInitHeight(height uint64) BlockOption {
-	return func(block *Blocks) error {
-		return block.storeInitHeight(height)
-	}
-}
-
 type Blocks struct {
-	store       *Storage
-	mux         sync.RWMutex
+	store *Storage
+	mux   sync.RWMutex
+	// todo LRU caching with size limit
 	heightCache map[byte]uint64
```

Comment:
Are we not interested in persisting this mapping?

Reply:
Yeah is just supposed to be a cache for quick access, but still persists. The cache should be improved to not just be a map tho, because it will keep growing.

Reply:
Ah, my bad, I missed that part.. All good :+1: